src/Utils/dialogUtils.tsx

```diff
@@ -283,6 +284,131 @@ export function getBestAction(scoreResponse: CLM.ScoreResponse, allActions: CLM.
     return best
 }
 
+// Return first unscored action that is available
+export function getBestUnscoredAction(scoreResponse: CLM.ScoreResponse, actions: CLM.ActionBase[], entities: CLM.EntityBase[], memories: CLM.Memory[]): CLM.ScoredAction | undefined {
+    if (scoreResponse.unscoredActions.length === 0) {
+        return undefined
+    }
+    // Return first available unscored action
```

Comment:
For my own understanding, what's the rationale behind returning the first unscored action here ?

Reply:
We have no scores, so we don't know which one would be best.  Picking the first is as good as any other.  Added a comment